xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -88,20 +97,26 @@ def _should_skip_target(*, ctx, target):
         target: The `Target` to check.
 
     Returns:
-        `True` if `target` should be skipped for target generation.
+        A `tuple` with two elements:
+
+        -   A `bool` indicating if `target` should be skipped for target generation.
+        -   The `skip_type` for this target.
```

Comment:
No need for the tuple. Can just return the `skip_type` if it should be skipped, or `None` otherwise.

Reply:
Right, I guess if there is a skip type it should indeed be what implies a skip is needed, ok I'll update this

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2196/commits/892ecded45235c59f6c316480d3ca733e3b1242e